src/hotspot/share/gc/shared/barrierSetNMethod.hpp

```diff
@@ -28,6 +28,9 @@
 #include "memory/allocation.hpp"
 #include "utilities/globalDefinitions.hpp"
 #include "utilities/sizes.hpp"
+#if INCLUDE_JVMCI
+#include "utilities/formatBuffer.hpp"
+#endif
```

Comment:
Given that this doesn't include any JVMCI files I think we can skip the INCLUDE_JVMCI guard (and sort in the added include). Or maybe better, forward declare FormatBuffer?